os/include/sys/types.h

```diff
@@ -66,6 +66,8 @@
 #include <semaphore.h>
 #endif
 
+#define CONFIG_RAM_END (CONFIG_RAM_START + CONFIG_RAM_SIZE)
```

Comment:
This must not be in types.h file. 
Please refer to nuttx code and fix this in tools/mkconfig.h